api/src/main/java/org/pmiops/workbench/cdr/CdrVersionService.java

```diff
@@ -70,56 +69,24 @@ public void setCdrVersion(DbCdrVersion version) {
     CdrVersionContext.setCdrVersionNoCheckAuthDomain(version);
   }
 
-  /**
-   * Sets the active CDR version, after checking to ensure that the requester is in the appropriate
-   * authorization domain. If you have already retrieved a workspace for the requester (and thus
-   * implicitly know they are in the authorization domain for its CDR version), you can instead just
-   * call {@link CdrVersionContext#setCdrVersionNoCheckAuthDomain(DbCdrVersion)} directly.
-   *
-   * @param cdrVersionId
-   */
-  public void setCdrVersion(Long cdrVersionId) {
-    this.setCdrVersion(
-        cdrVersionDao
-            .findById(cdrVersionId)
-            .orElseThrow(
-                () ->
-                    new NotFoundException(
-                        String.format("Cdr version %s does not exist", cdrVersionId))));
-  }
-
-  /**
-   * Sets the active CDR version, after checking to ensure that the requester is in the appropriate
-   * authorization domain. If you have already retrieved a workspace for the requester (and thus
-   * implicitly know they are in the authorization domain for its CDR version), you can instead just
-   * call {@link CdrVersionContext#setCdrVersionNoCheckAuthDomain(DbCdrVersion)} directly.
-   *
-   * @param cdrVersionId
-   */
-  public DbCdrVersion findAndSetCdrVersion(Long cdrVersionId) {
-    DbCdrVersion dbCdrVersion =
-        cdrVersionDao
-            .findById(cdrVersionId)
-            .orElseThrow(
-                () ->
-                    new NotFoundException(
-                        String.format("Cdr version %s does not exist", cdrVersionId)));
-    this.setCdrVersion(dbCdrVersion);
-    return dbCdrVersion;
-  }
-
   public Optional<DbCdrVersion> findByCdrVersionId(Long cdrVersionId) {
     return Optional.ofNullable(cdrVersionDao.findByCdrVersionId(cdrVersionId));
   }
 
   public CdrVersionTiersResponse getCdrVersionsByTier() {
-    List<DbAccessTier> tiers = accessTierService.getAccessTiersForUser(userProvider.get());
-    if (tiers.isEmpty()) {
+    boolean hasRegisteredTierAccess =
```

Comment:
we may be able to relax this condition as well.  But let's be conservative for now.